tests/deployment_test.go

```diff
@@ -210,34 +231,27 @@ func (d *DeploymentSuite) TestGetCurrentDeployment_NoCurrentDeployment() {
 	}
 }
 
+// verifyListDeployments makes a ListDeployment request and verifies that all the expected deployments eventually appear in the list.
+// Empty `seriesFilter` will list all deployments.
 func (d *DeploymentSuite) listDeployments(seriesName string, buildID string, withSeries bool) {
```

Comment:
function signature and the comment on top of it don't match. it seems this function does more than just listing. It's fine, if you want to keep that, maybe just make the name suggestive of that. e.g. `addDeploymentAndVerifyItIsListed`. (what I was suggesting before was to move the adding responsibility out of this and just keep the listing+verification part here).